src/components/drawer/ComponentDropOverlay.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+<div id="overlay">
+  <div class="overlay-background"></div>
+  <q-card id="text" rounded style="border-radius: 45px; background-color: white" flat>
```

Comment:
breakline
remove style and put it in class

Reply:
done